app/styles/ui/_autocompletion.scss

```diff
@@ -89,6 +89,8 @@
   flex-grow: 1;
   height: 100%;
   min-width: 0;
+	position: relative;
+	left: 6px;
```

Comment:
Hey @murrelljenna, I have to head out soon so forgive me if this is a premature review but I wanted to leave this feedback before I left for the day.

While the change you've done above works great we've historically avoided relative positioning unless we've really needed it, opting instead for padding. This is mostly a style preference without anything really substantial behind it but for no other reason that consistency I think we should do the same here.

Additionally we have [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) set up with the common spacing sizes we use throughout the app so that we can ensure consistent margins throughout the app. Our default spacer is 10px so In this case I'd use a half spacer which you'd use by writing something like this

```scss
padding: 0 var(--spacing-half)
```

This has the added benefit of applying the padding on both sides of the list item to ensure that long rows don't end up touching the right hand side border of the autocomplete popup.

If you're curious about what other variables you have at your disposal you can find them all in our [_variables.scss](https://github.com/desktop/desktop/blob/09bb5f34a6fac23334b0e3d02450fcf5ae80076d/app/styles/_variables.scss) file. Some of these (the ones related to color mostly) are also defined in the [dark theme](https://github.com/desktop/desktop/blob/09bb5f34a6fac23334b0e3d02450fcf5ae80076d/app/styles/themes/_dark.scss).

Reply:
No worries thanks for the comments, they're very helpful! I noticed a few of the variables when browsing and knowing now that theyre all in _variables.scss is good.

Should I make the changes youve suggested along with my indentation fix, or just the indentation and wait for more reviews?

Thanks again.

Reply:
Feel free to make both changes, we’re happy to review whenever you feel the changes are in a good place for it!